xrpl/asyncio/transaction/main.py

```diff
@@ -269,18 +269,13 @@ async def _check_fee(transaction: Transaction, client: Optional[Client] = None)
     Raises:
         XRPLException: if the transaction fee is higher than the expected fee.
     """
-    # Calculate the expected fee from the network load and transaction type
-    expected_fee = await _calculate_fee_per_transaction_type(transaction, client)
+    expected_fee = xrp_to_drops(0.1)  # a fee that is obviously too high
```

Comment:
Also doesn't AccountDelete have a much higher fee than this by default?

Reply:
Ok, I'll add it back in as a max